data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,413 @@ mission "Ringworld Debris: Quarg"
+			`	"We have even less information about how this system disappeared. If I may venture a guess, I'd say that the black hole is involved. The Builders never showed to possess the ability of making a system disappear, and we've seen in the past that some black holes can render inaccessible any hyperlane they are connected to given enough time. Perhaps this one, a young, non-rotating black hole, can do even more. There's a part of me that thinks this system could disappear again any moment, but if anything that motivates me to find as much as we can about it in the time we have" she says with a wink."`
+			choice
+				`	"Wait, are we in danger?"`
+				`	"How much will it take to finish processing the data?"`
+					goto dontcare
+			`	"Oh don't worry, I was just letting my imagination fly. There are thousands of other explanations that are as good as mine, none of which includes such a tragic ending." Suddenly, the computer Taely was working on earlier starts beeping. "Alright, let's see what it found," she says.`
+				goto beep
+			label dontcare
+			`	"Let me check." Soon after Taely starts walking towards the console, the computer she was working on earlier starts beeping. "Perfect, let's see what it found," she says.`
+			label beep
+			`	Taely starts browsing through a bunch of images on her console. In a few minutes, she appears to have selected half a dozen different events. "Look at these, <first>. What do you see?" The first image shows one of the Fetri'sei working on some kind of small structure. In the following pictures, presumably depicting different Fetri'sei, the structure becomes gradually bigger and more complex until the last one, where a Ka'het-like slug is shown growing inside this shell. "They are Von Neumann probes," she says. "Self replicating spacecraft. Unlike the Ka'het, which we found to be unable to reproduce and can only be created in a factory, these Ka'sei can duplicate their own structure with the material harvested from the local asteroids. The process probably takes years, or more, but it's still an incredible achievement."`
+			choice
+				`	"If they can duplicate themselves why aren't there more Ka'sei here?"`
+					goto kasei
+				`	"If the Builders were able to make self replicating spacecrafts why aren't the Ka'het like that as well?"`
+					goto kahet
+				`	"Do they have Jump Drives on them if they were able to get here?"`
+					goto jumpdrive
+				`	"Why aren’t they in other systems?"`
+					goto jumpdrive
+			label kasei
```

Comment:
It's only one option out of 3 that creates the loop, and I couldn't find a good way to avoid it

Reply:
I see no problem with having a loop when there aren't consequences to any of the conversational options. There's a variety of AAA franchises that consider loops to be a feature, not a problem, for non-time sensitive conversations (particularly ones where the player is asking questions about lore) as it allows the player to repeat a bit of conversation in case they missed something.